src/instrument/devices/data_management.py

```diff
@@ -10,18 +10,22 @@
     dm_workflow
 """.split()
 
+
 import logging
 
 from ophyd import Signal
 
+from ..dm.aps_data_management import dm_api_proc
```

Comment:
With apstools v1.6.20, much (all?) of this content is available from apstools.devices and apstools.utils

Reply:
I agree. I feel that moving to all existing apstools utilities deserves its own dedicated issue/pr